amt/middleware/route_logging.py

```diff
@@ -12,8 +16,35 @@
 
 class RequestLoggingMiddleware(BaseHTTPMiddleware):
     async def dispatch(self, request: Request, call_next: RequestResponseEndpoint) -> Response:
-        # todo(berry): embed Request object in logging library
-        logger.debug(f"Request: {request.method} {request.url}")
+        request_time = time()
+        request_id: str = str(ULID())
+        masker = Mask(mask_keywords=["cookie"])
+        request.state.request_id = request_id
+
         response = await call_next(request)
-        logger.debug(f"Response: {response.status_code}")
+        response_time = time()
+
+        response.headers["X-API-Request-ID"] = request_id
```

Comment:
This header is always added, also with static resources. Not a problem, but also not sure if it is needed.

Reply:
i do not think that this is an issue